main.js

```diff
@@ -59,12 +59,12 @@ const options = yargs
     describe: "Sets the hostname of the subject alternate name",
     type: "string"
   })
-  .check(function(argsv) {
-    if (argsv.type === "pfx") {
-      if (argsv.combined) {
-        throw new Error("Cannot use combine flag on PFX type certificate");
-      } else if (argsv.password == null || argsv.password.length == 0) {
-        throw new Error("Please enter a valid password for this certificate");
+  .check(function(argv, options) {
+    if (argv.type.toLowerCase() === "pfx") {
+      if (argv.combined) {
+        throw new Error("Cannot use combine flag on PFX certificate");
+      } else if ( argv.password === undefined || argv.password.length == 0) {
```

Comment:
I'm pretty sure this whole line can become:
```js
else if ( !argv.password)
```

Here is a fiddle to prove it: https://jsfiddle.net/htxad0j2/